src/Fantomas.Tests/TokenParserTests.fs

```diff
@@ -658,3 +658,29 @@ let isUnix =
 
     getDefines source
     == [ "NETSTANDARD1_6"; "NETSTANDARD2_0" ]
+
+
+
+[<Test>]
```

Comment:
Please move this test to CompilerDirectiveTests.fs, it fits better over there.
Would you also mind adding the issue number in the test name? See [guidelines](https://github.com/fsprojects/fantomas/blob/master/CONTRIBUTING.md#guidelines).